portfolio/src/main/webapp/script.js

```diff
@@ -12,12 +12,28 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
+/** Retrieves data from the /data page and displays it. */
+function getComments() {
+  fetch('/data').then(response => response.json()).then((json) => {
+    json.forEach(commentText => displayComment(commentText));
+  });
+}
+
+window.addEventListener('DOMContentLoaded', getComments, false);
```

Comment:
Nit: move this line to the bottom of the file, top of the file, or at least below `displayComment`.

`getComments` and `displayComments` are closely related (one calls the other) and should be next to each other in the file.

Reply:
Done.